doc/en/changelog.rst

```diff
@@ -266,6 +266,11 @@ These are breaking changes where deprecation was not possible.
   therefore fail on the newly-re-emitted warnings.
 
 
+- The internal ``FixtureManager.getfixtureclosure`` method introduced a breaking change (mostly by accident):
+  the ``ignore_args`` parameter was changed to an ``AbstractSet[str]`` and lost the default argument. Plugins
+  which subclass ``FixtureManager`` and overwrite that method will need to adapt to the change.
```

Comment:
I think it would be too complex to explain all of the changes to the function, so I think we should basically only say "it broke", e.g.

```suggestion
- The internal ``FixtureManager.getfixtureclosure`` method has changed. Plugins which use this method or
  which subclass ``FixtureManager`` and overwrite that method will need to adapt to the change.
```

Reply:
Agreed, thanks!